src/repository/participant-repository.ts

```diff
@@ -0,0 +1,74 @@
+import { Participant } from "../model/participant";
+import { ParticipantParams } from "../util/types";
+import { Company } from "../model/company";
+
+export class ParticipantRepository {
+  async createParticipant(
+    participantParams: ParticipantParams,
+    companyId: number
+  ) {
+    try {
+      const company = await Company.findOne({
+        where: { company_id: companyId },
+      });
+      if (company) {
+        return await Participant.create({
+          name: participantParams.name, 
+          phone_number: participantParams.phone_number,
+          email: participantParams.email,
+          company: company,
+        }).save();
+      }
+    } catch (err) {
+      console.error(err);
+    }
+  }
+
+  async deleteParticipant(participantId: number) {
+    try {
+      const participant = await Participant.findOne({
+        where: {
+          participant_id: participantId,
+        },
+      });
+      if (participant) {
+        return await Participant.remove(participant);
+      }
```

Comment:
you can put else and throw error that participant doesn't exist.

Reply:
Fixed. Also added additional error messages.